### ISSUE TITLE
Behavior of dehydrate

### ISSUE THREAD

**Issue by jmeas** [mattkrick/fast-bitset#3]
The docs describe `dehydrate` as...

> Turn the bitset into a comma separated string that **skips trailing 0 words**...

[The tests](https://github.com/mattkrick/fast-bitset/blob/0a317928a3a202d1c5c4e355aae841f69558a63c/spec/BitsetSpec.js#L22-L32) don't seem to be skipping them, unless I'm misinterpreting what "skip" means?

I would have expected

```js
expect(bs.dehydrate()).toBe('0,1,99');
expect(bs.dehydrate()).toBe('2147483647,99');
```

based on the README.

Am I just misunderstanding? :)

**Comment by mattkrick** [comment]
good catch, will fix tonight

**Comment by mattkrick** [comment]
fixed with 3b9a2c85c15eb66186dbcfcb7d7c45a15c0d9e03
